examples/02-plot/vertices.py

```diff
@@ -1,10 +1,8 @@
-"""
-Visible Vertices
-~~~~~~~~~~~~~~~~
+"""Visible Vertices
+~~~~~~~~~~~~~~~~.
```

Comment:
Seems that it doesn't do a great job with titles for these examples. We can "cheat" by providing a reference at the top. Probably a good idea anyway.

```suggestion
""".. _visible_vertices_example:

Visible Vertices
~~~~~~~~~~~~~~~~.

```

Reply:
```suggestion
""".. _vertices_example:
Visible Vertices
~~~~~~~~~~~~~~~~
```